src/E3_indicator_calculation.py

```diff
@@ -9,3 +9,4 @@ def all_totals(dict_values):
             {column: dict_values["kpi"]["cost_matrix"][column].sum()}
         )
     return
+
```

Comment:
for forgotten commits, stage the change, then use

`git commit --amend` to add them to the previous commit (you also have the possibility to modify the commit message)

(only work for the last commit and you need to force push afterwards `git push -f`)

Reply:
Thanks! I was just trying around a bit to see if the :math: equations where at least working online, so it is a bit messy.